app/controllers/api/v1/assignments_controller.rb

```diff
@@ -2,7 +2,9 @@ module Api
   module V1
     class AssignmentsController < ApplicationController
       include CanvasValidationHelper
+      
       before_action :validate_ids!, only: [:create]
+      skip_before_action :verify_authenticity_token
```

Comment:
What was the reason for this?  This method checks the CSRF token and Rails suggests it not be subverted for any reason.  In the case that issues arise, they recommend rewriting it to suit the use case.

Reply:
Addressed in PR body.  Note that removing this still allows the tests to pass, though integration may not follow suit.